libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, ElementRef, Inject, Input, LOCALE_ID, ViewChild } from '@angular/core';
+
+import { Options } from 'highcharts';
+import * as Highcharts from 'highcharts/highstock';
+import AnnotationsModule from 'highcharts/modules/annotations';
+
+import { annotations, HighstockDataPoint } from './options/highstock-chart-options';
+
+// @ts-ignore
+AnnotationsModule(Highcharts);
+
+@Component({
+  selector: 'kirby-highstock-chart',
+  templateUrl: './highstock-chart.component.html',
+  styleUrls: ['./highstock-chart.component.scss'],
+})
+export class HighstockChartComponent {
+  private _data: HighstockDataPoint[];
+  @Input() set data(val: HighstockDataPoint[]) {
+    this.onDataChanges(val);
+  }
+  get data() {
+    return this._data;
+  }
+  private _options: Options;
+  @Input() set options(val: Options) {
+    this.onOptionsChanges(val);
+  }
+  get options() {
+    return this._options;
+  }
+  private _height = 300;
+  @Input() set height(height: number) {
+    this._height = height;
+    if (this.chart) {
+      this.chart.setSize(null, height);
+    }
+  }
+  get height() {
+    return this._height;
+  }
+  @Input() description = '';
+  @Input() showDataLabels = true;
+
+  @ViewChild('chartContainer', { static: true }) chartContainer: ElementRef;
+
+  chart: Highcharts.Chart;
+
+  constructor(@Inject(LOCALE_ID) private locale: string) {}
+
+  onOptionsChanges(options: Options) {
+    this._options = options;
+    this.chart = Highcharts.stockChart(this.chartContainer.nativeElement, this._options);
```

Comment:
Consider only importing what's used:
```suggestion
    this.chart = stockChart(this.chartContainer.nativeElement, this._options);
```

Reply:
Done.